cost/google/instances_stackdriver_utilization/README.md

```diff
@@ -1,36 +1,51 @@
-## Google Instance StackDriver Utilization Policy
+# Google Inefficient Instance Utilization using StackDriver
 
-### What it does
+## What it does
 
-This Policy Template gathers Google StackDriver utilization for instances on 30 day intervals. This is meant to be run as a monthly policy. 
+This Policy Template gathers Google StackDriver utilization for instances on 30 day intervals and resizes them after user approval. This is meant to be run as a monthly policy.
```

Comment:
please remove This is meant to be run as a monthly policy.

Reply:
Updated